KNXEngine/dptlib/dpt60001.js

```diff
@@ -0,0 +1,272 @@
+  }
+}
+
+function griesserPrio (prio, command) {
+  const prioCommand = ((command & 224) / 32) >> 0
+  if ((((prio & 252) / 4) >> 0) === 0) {
+    switch (prioCommand) {
+      case 0: return 'border command'
+      case 1: return 'automatic command'
+      case 3: return 'priority command'
+      case 4: return 'warning command'
+      case 5: return 'security command'
+      case 6: return 'danger command'
+      default: return 'unknown priority' + prioCommand
+    }
+  } else {
+    return '-'
+  }
+}
+
+// Send to BUS
+exports.formatAPDU = function (value) {
+  if (!value) {
+    knxLog.get().error('DPT60001: cannot write null value')
+  } else {
+    if (typeof value === 'object' &&
+    Object.prototype.hasOwnProperty.call(value, 'command') &&
+    Object.prototype.hasOwnProperty.call(value, 'data') &&
+    Object.prototype.hasOwnProperty.call(value, 'sectors') &&
+    value.data[0] === 'localoperation'
+    ) {
+      const sectorCode = griesserSectorToSectorCode(value.sectors)
+      const commandCode = griesserCommandToCommandCode(value.command)
+      const p1 = griesserCommandToCommandCodeP1(value.data[1])
+      const bufferTotal = Buffer.alloc(6)
+      bufferTotal[0] = parseInt(toRadix(sectorCode, 2).slice(-8), 2)
+      bufferTotal[1] = parseInt(toRadix(commandCode, 2).slice(-6) + toRadix(sectorCode, 2).slice(-10, -8), 2)
+      bufferTotal[2] = parseInt(toRadix(p1, 2).slice(-8), 2)
+      return bufferTotal
+    } else {
+      knxLog.get().error('DPT60001: Must supply an value {command:"operation code", data:["localoperation", "long up"], sectors:[159]}')
+    }
+  }
+}
+
+// RX from BUS
+exports.fromBuffer = function (buf) {
+  if (buf.length !== 6) {
+    knxLog
+      .get()
+      .warn(
+        'DPTGriesser.fromBuffer: buf should be 6 bytes long (got %d bytes)',
+        buf.length
+      )
+    return null
+  } else {
+    const hexToDecimal = (hex) => parseInt(hex, 16)
+    const bufTotale = buf.toString('hex')
+    const Byte0 = hexToDecimal(bufTotale.slice(0, 2))
+    const Byte1 = hexToDecimal(bufTotale.slice(2, 4))
+    const Byte2 = hexToDecimal(bufTotale.slice(4, 6))
+    const Byte3 = hexToDecimal(bufTotale.slice(6, 8))
+    const Byte4 = hexToDecimal(bufTotale.slice(8, 10))
+    const Byte5 = hexToDecimal(bufTotale.slice(10, 12))
+    const sectorCode = griesserSectorCode(Byte0, Byte1)
+    const commandCode = griesserCommandCode(Byte1, Byte2)
+
+    return {
+      Byte0,
+      Byte1,
+      Byte2,
+      Byte3,
+      Byte4,
+      Byte5,
+      sectorCode,
+      commandCode,
+      sectors: griesserSectors(sectorCode),
+      prio: griesserPrio(Byte1, Byte2),
+      command: griesserCommand(commandCode),
+      data: griesserParameter(commandCode, Byte2, Byte3, Byte4, Byte5)
+    }
+  }
+}
+
+// DPT Griesser Object basetype info
+exports.basetype = {
+  bitlength: 4 * 8 + 2 * 6 + 1 * 10,
+  valuetype: 'composite',
+  desc: 'Commands for solar shading actors',
+  help: ''
```

Comment:
Please add a payload sample in the "help" property, to help users understand how to handle this datapoint. This is what i wrote for DPT213:

help:
        `// Sample of 213.100.
// Set the temperatures between -272°C and 655.34°C with 0,02°C of resolution.
// These 4 property names, are valid for 213.101, 213.102 etc... as well.
// For example, for 213.101, LegioProtect is the "Comfort" property, Normal is "Standby", etc...
msg.payload = {Comfort:21.4, Standby:20, Economy:18.2, BuildingProtection:-1};
return msg;`